inc/usb_ccid.h

```diff
@@ -0,0 +1,71 @@
+#include <stdint.h>
+
+#ifndef _USB_CCID_H_
+#define _USB_CCID_H_
+#ifdef __cplusplus
+    extern "C" {
+#endif
+
+#define USB_CLASS_CCID                   0x0b    /**<\brief Smart Card Device class */
+#define USB_DTYPE_FUNCTIONAL_DESC   0x21   /**<\brief Funcitonal descriptor.*/
+#define USB_DTYPE_CCID_FUNCTIONAL   0x21   /**<\brief Functional descriptor.*/
+
+/** Endpoint size in bytes of the CCID data being sent between IN and OUT endpoints. */
+#define CCID_EPSIZE          64
```

Comment:
According to the specification, the endpoint size is a flexible value that depends on configuration. Do you really need to bind it?

Reply:
No - I can remove it as well.